tests/fixtures/metadata.py

```diff
@@ -965,3 +966,41 @@ def templ_patron_public_martigny(
         reindex=True)
     flush_index(TemplatesSearch.Meta.index)
     return template
+
+
+# --- LOCAL FIELDS
+@pytest.fixture(scope="module")
+def local_field_1_data(local_fields):
+    """Load Local field 1 data."""
+    return deepcopy(local_fields.get('lofi1'))
+
+
+@pytest.fixture(scope="module")
+def local_field_1(app, org_martigny, document, local_field_1_data):
```

Comment:
if possible rename:
 `local_field_1` to `local_field_martigny`
 `local_field_2` to `local_field_sion`

this way works too, only if you have the time

Reply:
done